### ISSUE TITLE
rename AccessibleNumberTweaker to AccessibleNumberSpinner

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#386]
While reviewing scenery-phet components for https://github.com/phetsims/sun/issues/121#issuecomment-399162547, I ran across `AccessibleNumberTweaker`, which is currently mixed into `NumberPicker` and `NumberSpinner`, and (somewhat surprisingly) not associated with anything named "tweaker". 

Highly recommended to rename this to `AccessibleNumberSpinner`.   The terms "tweaker" and "picker" are PhET inventions, while "spinner" is the generally-accepted term for a UI component that changes a value using arrow buttons.

**Comment by jessegreenberg** [comment]
> he terms "tweaker" and "picker" are PhET inventions, while "spinner" is the generally-accepted term for a UI component that changes a value using arrow buttons.

I am convinced, I will go ahead with the name change.

**Comment by jessegreenberg** [comment]
Rename done in https://github.com/phetsims/scenery-phet/commit/f477a8577466d5ea50429e8d74dea59afbc88554 and https://github.com/phetsims/sun/commit/a1d1b42472b628d4a1c104f175b53b4b5cf35f8b. Closing  this issue, and removed all references to "tweaker" in documentation about this trait.
